### PR TITLE
feat(prettier-config): trailingComma to all and added missing options

### PR DESCRIPTION
**Pull request by lipis**
## Pull Request Checklist

- [ ] My code is covered by tests
- [ ] I will [merge the PR as breaking change](https://github.com/wireapp/wire-web-packages/wiki/Releases#create-major-release), if the API contract changes

### CODE REVIEW

File: packages/prettier-config/index.json [hunk 1]
```diff
@@ -3,15 +3,17 @@
...
+  "jsxSingleQuote": false,
   "printWidth": 120,
   "proseWrap": "never",
   "quoteProps": "as-needed",
   "requirePragma": false,
   "semi": true,
   "singleQuote": true,
   "tabWidth": 2,
-  "trailingComma": "es5",
+  "trailingComma": "all",
```
Comment (lipis) [inline packages/prettier-config/index.json]: Can someone remind me why we didn't have `all` here?

Comment (ffflorian) [reply packages/prettier-config/index.json]: Because it's annoying to have commas in function arguments :)

**Comment by lipis** [comment]
That's why I opened it... to talk :)

File: packages/prettier-config/index.json (hunk 1 above)
Comment (lipis) [reply packages/prettier-config/index.json]: Define "annoying" :) Can you give me an example on why it's annoying to have the comma there?

Comment (lipis) [reply packages/prettier-config/index.json]: https://vote4code.com is still up and running.. 🗡

File: packages/api-client/src/main/APIClient.test.node.js [hunk 2]
```diff
@@ -215,7 +215,7 @@ describe('APIClient', () => {
       const context = client.createContext(
         '3721e5d3-558d-45ac-b476-b4a64a8f74c1',
         'temporary',
-        'dce3d529-51e6-40c2-9147-e091eef48e73'
+        'dce3d529-51e6-40c2-9147-e091eef48e73',
```
Comment (lipis) [inline packages/api-client/src/main/APIClient.test.node.js]: consistency.. :)

**Pull request merged**